### DISCUSSION TITLE
github actions中出现如下问题，请问谁知道原因，将github升级到了plus版依然无效，该仓库为私有仓库

### DISCUSSION THREAD

**Discussion by fangyalong** [community/community#75583] [New to GitHub]
![1700011258831](https://github.com/community/community/assets/111187295/86af9b6e-63fe-475a-aaaf-0caa5d61a077)

**Comment by queenofcorgis** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
